### DISCUSSION TITLE
tine podcast:  06 – Der Dateimanager

### DISCUSSION THREAD

**Discussion by pschuele** [tine20/tine20#7442] [General]
Diese Episode des tine Podcast beschäftigt sich mit einer einzelnen Anwendung in tine: dem Dateimanager. In den vergangenen fünf Jahren ist der Dateimanager so sehr gewachsen, dass es sich lohnt mal einen Blick in die Kontextmenüs und Konfigurationen zu werfen. Einige Funktionen (aber immer noch nicht alle) erläutert Philipp Schüle, Senior Core Entwickler von tine.

https://www.tine-groupware.de/podcast/06-der-dateimanager/

**Comment by lab-at-nohl** [comment]
Grundsätzlich fände ich es toll, wenn es keine reine Feature Besprechung wäre. Ab und an mehr Technik, wie was wo in die config muss. Wie die Übergabe an den Viren Scanner erfolgt usw. 

Wie aktiviere ich z.B. die Versionen und stelle die Zahl der vorgehaltenen Versionen ein?

**Reply by pschuele** [reply]
stimmt, das war eine recht feature-lastige folge ... ich denke, das ist nicht die regel.

zur technik ->

der virenscanner wird so aktiviert (z.b. in config.inc.php):

~~~php
     'filesystem' => [
        'avscanURL' => 'tcp://clamd.uri:3310',
        'avscanMode' => 'quahog',
        'maxFSize' => 524288000, // 500MB
        // ...
     ],
~~~

dateimanager-versionenierung:

~~~php
     'filesystem' => [
         'modLogActive' => true,
         'numKeepRevisions' => 3, // number of revisions to keep
         'monthKeepRevisions' => 3, // number of months revisions being kept before they are automatically deleted
         // ...
     ],
~~~

=> die konfigurationsoptionen findet man auch in der Tinebas/Config.php beschrieben.
